tool/processors/collectd/collectd.go

```diff
@@ -21,7 +21,7 @@ func (p *processor) Process(ctx *runtime.Context, config *data.Config) {
 	if ctx.OsParameter == util.OsTypeWindows {
 		return
 	}
-	yes := util.Yes("Do you want to monitor metrics from CollectD?")
+	yes := util.Yes("Do you want to monitor metrics from CollectD? Note that depending on the system, some require the CollectD software to be installed on your server (e.g. Amazon Linux 2) or the /usr/share/collectd/types.db file to be created.")
```

Comment:
It would be important not to break existing user workflows such as causing the config generator to fail if collectd is not installed properly.  Someone could config the agent ahead of installing collectd.

What's the current agent behavior if collectd is configured but not installed?

Reply:
The Agent fails to start:
https://github.com/awsdocs/amazon-cloudwatch-user-guide/issues/54

Reply:
Hi all, thanks for the review, I think the recommendation @adam-mateen made for option 1 fits the length (there's a longer prompt from `func wantEC2TagDimensions`) and is sufficient for the user to be aware of aware of the CollectD installation. Shall I proceed with the refactor for option 1?

Reply:
I think the wording is sufficient. I'd say go for it